### DISCUSSION TITLE
Changing username

### DISCUSSION THREAD

**Discussion by brengy** [community/community#146458] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

Actions

### Body

how to change username
example.github.io/
change 'example' to another one

**Comment by Zul-Qarnain** [comment]
To change your GitHub username and update your GitHub Pages URL accordingly, follow these steps:
---
### **Steps to Change Your GitHub Username**

1. **Sign in to Your [GitHub](https://github.com/) Account**:
   - Visit [GitHub](https://github.com) and log in.

2. **Access Your Account Settings**:
   - Click on your profile picture in the top-right corner.
   - Select **Settings** from the dropdown menu.

3. **Change Your Username**:
   - In the **Account Settings** page, click **Edit** next to your current username.
   - Enter your new desired username. GitHub will check if it is available.
   - Confirm the change by following any additional instructions.

4. **Update GitHub Pages URL**:
   - When you change your username, your GitHub Pages URL will also change automatically. For example:
     ```
     example.github.io → newusername.github.io
     ```
   - No additional action is required to update the GitHub Pages site itself. GitHub handles the redirect.

---
If you encounter issues during this process, let me know!

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @brengy ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you: Pages. Good luck!

**Comment by mecodeatlas** [comment]
Hi @brengy 👋🏾,It looks like @Zul-Qarnain's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)

**Comment by achewyc0okie** [comment]
change my username
